unittests/Feature/Feature.cpp

```diff
@@ -4,7 +4,7 @@
 
 namespace vara::feature {
 TEST(BinaryFeature, getName) {
-  BinaryFeature F("Test", false);
+  BinaryFeature F("Test", false, std::nullopt);
```

Comment:
```suggestion
  BinaryFeature F("Test", false);
```
Could be remove when adapting the ctor behavior.